site/content/api/Tenant-API.md

```diff
@@ -269,7 +270,7 @@ Hono uses the following AMQP message delivery states when receiving request mess
 
 Most of the operations of the Tenant API allow or require the inclusion of tenant data in the payload of the
 request or response messages of the operation. Such payload is carried in the body of the corresponding AMQP 
-messages as part of a single *AMQP Value* section.
+messages as part of a single *Data* section. The content type must be set to `application/json`.
```

Comment:
IMHO we should explicitly define the content-type property as mandatory in the definition of each operation that requires such payload. We also define the subject there explicitly ...

Reply:
I did add this to the "standard" section of each API, as this affects all messages.